src/Shared/CoreCLRAssemblyLoader.cs

```diff
@@ -63,11 +72,29 @@ public Assembly LoadFromPath(string fullPath)
             }
         }
 
+        private Assembly TryGetWellKnownAssembly(AssemblyLoadContext context, AssemblyName assemblyName)
```

Comment:
I find a method named `Try...` that doesn't return a `bool` a bit surprising in C#.